articles/how-to-fix-and-prevent-sql-injection-in-wordpress/index.md

```diff
@@ -0,0 +1,194 @@
+
+#### WordPress Plugins Vulnerability
+
+WordPress is the most popular open-source Content Management System (CMS) running millions of websites. It's approximated the [CMS enjoys a 61% market share](https://kinsta.com/blog/wordpress-statistics/). Your WordPress website is secure from SQLi only if the WordPress core files are up-to-date.
+
+##### What are WordPress Core files?
+
+This are the files that make up the open-source CMS project. Below is an image of what [WordPress core files](https://wpsupergeek.com/what-are-the-wordpress-core-files/#:~:text=When%20you%20install%20WordPress%2C%20you,admin%20interface%20of%20your%20website.&amp;text=Core%20files%20make%20up%20the,themes%2C%20plugins%2C%20and%20images.) look-like.
+![wordpress-core-files](/engineering-education/how-to-fix-and-prevent-sql-injection-in-wordpress/wordpress-core-files.png)
+Ask yourself, is WordPress Core running on the updated patch?
+
+As we have found out, your WordPress site is secure from any SQLi if your core files are updated. Nevertheless, use of third-party plugins and themes change the narrative and exposes you exponentially. We shall list the most prone WordPress plugin installation areas that you be on the look-out. They include;
+
+1. Subscription pop-ups
+2. Login/Signup forms
+3. E-commerce checkout pages/carts
+4. Search bars
+5. Contact forms
+6. Feedback forms
+7. Generic contact forms
+8. Blog comment forms
+9. Search parameters
+
+The [Exploit-DB website](https://www.exploit-db.com/search?q=) provides a detailed context of vulnerabilities happening in WordPress plugins. Furthermore, it points out how the attacks affected victim websites and the solution deployed.  
+
+### How to Detect and Fix WordPress SQLi Attack?
+
+##### Fix WordPress SQLi Attack: Damage Control
+
+An attacker targets the database at any given point as that is the only source for information on your website. Firstly, disable admin privileges on your database. Henceforth, the attacker is tied down a read-only database that is not of much help to them.
+
+Proceed to sanitize the UDF files the attacker used to obtain the OS Shell. In addition, change your encrypted password to more stronger values. Finally, check if the port 3306 is open on the internet and if so, block it.
+
+##### Fix WordPress SQLi Attack: Database Clean Up
+
+A database clean-up is required to fix any possible SQLi on your site's database. Here are steps to so as well with the function cheat-sheet.
```

Comment:
to **do** so as well **as** the function cheatsheet would make more sense.

Reply:
Corrected.